src/components/notifications/YZJ.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="mb-3">
+        <label for="yzjWebHookUrl" class="form-label">{{ $t("YZJ Webhook URL") }}<span style="color: red;"><sup>*</sup></span></label>
+        <input id="yzjWebHookUrl" v-model="$parent.notification.yzjWebHookUrl" type="url" class="form-control" required />
+        <i18n-t class="form-text" keypath="wayToGetTeamsURL">
+            <a href="https://www.yunzhijia.com/opendocs/docs.html#/tutorial/index/robot" target="_blank">{{ $t("here") }}</a>
+        </i18n-t>
+    </div>
+    <div class="mb-3">
+        <label for="yzjType" class="form-label">{{ $t("YZJ Robot Type") }}<span style="color: red;"><sup>*</sup></span></label>
```

Comment:
what does this field mean?
Could you please add a helptext to clarify this?

Also:
if there is truely only one option for this value, we don't need a selection and can remove it.

Reply:
There can be multiple values, but I've only implemented one of them so far